waku/v2/node/wakunode2.nim

```diff
@@ -392,9 +392,11 @@ proc info*(node: WakuNode): WakuInfo =
   ## Status: Implemented.
   ##
 
-  # TODO Generalize this for other type of multiaddresses
   let peerInfo = node.peerInfo
-  let listenStr = $peerInfo.addrs[^1] & "/p2p/" & $peerInfo.peerId
+  var listenStr = ""
+  for address in node.announcedAddresses:
+    var fulladdr = "[" & $address & "/p2p/" & $peerInfo.peerId & "]" 
+    listenStr &= fulladdr
   let wakuInfo = WakuInfo(listenStr: listenStr)
```

Comment:
Considering this is used in WakuInfo response, e.g. here https://rfc.vac.dev/spec/16/#wakuinfo It might make sense to make this response more programmatic. I.e. if we return multiple addresses, it should probably return a proper sequence as opposed to a constructed string. I don't know what other libp2p users do here but I'm sure others have the equivalent of WakuInfo somewhere? Ethereum, IPFS, etc. Could be pulled out as an issue but probably not high priority.

However, this is better than what exists right now and would be useful for logs, so seems fine to me.